datasets/UKNI01_northern_ireland/graph_values.yml

```diff
@@ -170,36 +174,50 @@ energy_heat_burner_wood_pellets:
   demand: 0.0
   output:
     :steam_hot_water: 0.9
+  full_load_hours: 2190.0
 energy_chp_supercritical_waste_mix:
   demand: 885.1851851851851
   full_load_hours: 4155.0
   output:
     :steam_hot_water: 0.15
     :electricity: 0.27
 energy_chp_ultra_supercritical_lignite:
-  demand: 0.0
```

Comment:
I don't think this line should be deleted: this would lead to an earlier issue where non-existent lignite power plants were added to the model.

Reply:
Good points (this and the next!). @antw do you know why this demand is deleted? We have added `demand` to the `graph_methods` in the nodes ànd added a sparse graph query setting this demand

Reply:
The demand values aren't deleted: they're just moved a few lines lower.

![Screenshot 2022-02-02 at 14 30 54](https://user-images.githubusercontent.com/4383/152173812-1ccb2e29-42f8-4bac-8d37-02f56089e5b8.png)

Reply:
Woooooops! Sorry

Reply:
That one is on me, sorry!